integrand-py/tests/test_integrand.py

```diff
@@ -139,6 +144,39 @@ def test_delete_topic(self):
         integrand.CreateTopic(topicName)
         response = integrand.DeleteTopic(topicName)
         assert response['status'] == 'success'
+        
+    def test_delete_topic_used_by_workflow_raises_exception(self, workflow_function_name):
```

Comment:
Okay nice, so this tests that we have to delete the workflow first before deleting the topic.

Great addition(s)